src/main/java/duke/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.command;
+
+import duke.util.Parser;
+import duke.util.TaskList;
+import duke.task.Deadline;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents the Command of adding a new deadline to a task list.
+ */
+public class AddDeadlineCommand extends Command {
```

Comment:
Should class names be nouns? Maybe can rename to something like `DeadlineCommand` here?